dialog/fileicon.go

```diff
@@ -11,41 +11,43 @@ import (
 )
 
 const (
-	fileIconSize = 64
-	fileTextSize = 24
+	fileIconSize      = 64
+	fileTextSize      = 24
+	fileIconCellWidth = fileIconSize * 1.25
 )
 
-type fileIcon struct {
+type fileDialogIcon struct {
```

Comment:
While this is a better name than the old one, I'm still confused by it because this is a complete widget while an icon is just some kind of resource IMO.
Maybe that's just my biased view?
I often name variables representing complex structures by their type like `win := Window.create()` or `dialog := NewDialog()`. Doing this with the `fileDialogIcon` and accessing its `icon` property would lead to stuttering:
```go
icon := fileDialogIcon{…}
if icon.icon != nil {
  …
}
```
I think an icon _is_ an icon and doesn't _have_ an icon :).

Reply:
Good point - how about fileDialogItem?